product_docs/docs/biganimal/release/getting_started/creating_a_cluster/01_cluster_networking.mdx

```diff
@@ -4,59 +4,33 @@ title: Cluster networking architecture
+### Standard SKU Load Balancer in Azure
 
-## Private cluster load balancing
+When a cluster is created with public network access, a load balancer is created and configured with a public IP address. Once assigned, this IP address does not change unless you change the networking configuration for your cluster. The load balancer always routes to the leader of your cluster. 
 
-When a cluster is created with private network access, an Azure
-Standard SKU Load Balancer is created and configured with an IP
-address that always routes to the leader of your cluster. Once
-assigned, this IP address does not change unless you change the
-networking configuration for your cluster.
+Only one load balancer is typically deployed in an Azure region. BigAnimal adds more IP addresses to the existing load balancer for subsequent clusters in the Azure region.
 
-This IP address is private to the VNet hosting your BigAnimal
-services. By default it is not routable from other networks, even in
-your Azure account. See [Setting up Azure infrastructure to connect to a private network cluster](../../using_cluster/02_connecting_your_cluster/#setting-up-azure-infrastructure-to-connect-to-a-private-network-cluster) for details and instructions
-on how to properly configure routing for private clusters.
+Every BigAnimal cluster, regardless of public or private networking status, is assigned a single DNS zone that maps to its exposed IP address, either public or private. When toggling between public and private, wait up to 120 seconds for DNS caches to flush.
 
-Only one Azure Internal Load Balancer per BigAnimal region is typically deployed in your
-Azure subscription. Subsequent private clusters add more IP addresses to the existing load balancer.
+Clusters can be changed from public to private and vice versa at any time. When this happens, the IP address previously assigned to the cluster is deallocated, a new one is assigned, and DNS is updated accordingly.
 
-## DNS
+### Amazon Network Load Balancer in AWS. 
+ 
+BigAnimal creates a new load balancer for each clusters and tags it with the ckuster ID in the following format: 
 
-Every BigAnimal cluster, regardless of public or private networking
-status, is assigned a single DNS zone that maps to its exposed IP
-address, either public or private. When toggling between public and
-private, wait up to 120 seconds for DNS caches to flush.
+service.k8s.aws/stack: default/*<cluster_ID>* (for example, service.k8s.aws/stack: default/p-c4j0jfcmp3af2ieok5eg).
 
-## Toggling between public and private
-
-Clusters can be changed from public to private and vice versa at any
-time. When this happens, the IP address previously assigned to the
-cluster is deallocated, a new one is assigned, and DNS is updated
-accordingly.
+Because the load balancer IP address used in AWS is dynamic, ensure that your application uses the correct DNS name to access the network load balancer of a particular cluster.
```

Comment:
This doesn't need to block release but there's some subtlety here that we'll need to work out. While the IP addresses are dynamic, unless I'm mistaken each zone's IP address is static. Some customers will want to know the full range of IP addresses that they _might need to connect to_, so that they can configure outbound firewall rules.

Reply:
@aittam Can you provide your input on this.

Reply:
I think even the IP address for each zone can change, at least that is definitely the case for Application Load Balancer. They add more IPs when they scale up and remove them when they scale down. It also changes when they do maintenance. I think the only way to be sure would be to use an EIP for each AZ. Since now we have business support we can also open a ticket and ask.